spec/controllers/api/v1/books_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+require 'shared/contexts.rb'
+
+describe Api::V1::BooksController, type: :controller do
```

Comment:
This test it's OK but... In general we use a static JSON schema to test against an api response. A JSON schema it's a file that describes the structure of a JSON. So this way if someone changes the serializer the test fails. 

Right now, changing the serializer will have no impact in your tests because the assert element you are using also changes.